### PR TITLE
Connext

### PR DESCRIPTION
**Pull request by pedrouid**
Integrates connext into the torus wallet. WIP.

### CODE REVIEW

**Comment by pedrouid** [comment]
Requires torusresearch/torus-website#526

File: src/channel-provider.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+export class TorusChannelRpcConnection extends SafeEventEmitter {
+  constructor(connectionStream) {
+    super()
+    this.promises = {}
+    this.connectionStream = connectionStream
+
+    const self = this
+    // super constructor
+    SafeEventEmitter.call(self)
```
Comment (chaitanyapotti) [inline src/channel-provider.js]: This isn't necessary. super() call exists

Comment (pedrouid) [reply src/channel-provider.js]: Yeah I thought that was weird but I was just following some pattern I saw somewhere else in the codebase

Comment (chaitanyapotti) [reply src/channel-provider.js]: We refactored that to follow es6 now. Please use super() only

Comment (pedrouid) [reply src/channel-provider.js]: I already fixed it on https://github.com/torusresearch/torus-embed/pull/65/commits/94c98808c59c74986ddfc486c544931998ea3e3d

**Comment by chaitanyapotti** [comment]
@pedrouid Please merge the current develop with your code

**Comment by pedrouid** [comment]
@chaitanyapotti resolved merge conflicts

**Pull request merged**